integration-tests/slack/circleci-utils.js

```diff
@@ -13,17 +20,21 @@ const fetchJobArtifacts = async ({ buildNum = process.env.CIRCLE_BUILD_NUM } = {
     throw new Error('**  ERROR: Missing CircleCI build number. Failed to fetch CircleCI job artifacts.');
   }
 
-  // For more arguments and details of the response, see: https://circleci.com/docs/api/v2/index.html#operation/getJobArtifacts
-  const apiUrlRoot = 'https://circleci.com/api/v2/project/github/DataBiosphere/terra-ui';
+  // The v1.1 artifacts API does not need token, so we use it instead.
+  const apiUrlRoot = 'https://circleci.com/api/v1.1/project/github/DataBiosphere/terra-ui';
   try {
     // Because terra-ui is a public repository on GitHub, API token is not required. See: https://circleci.com/docs/oss#security
     const response = await fetch(`${apiUrlRoot}/${buildNum}/artifacts`);
-    const { items } = await response.json();
+    checkStatus(response);
+
+    const items = await response.json();
     const testSummaryArtifacts = _.filter(_.flow(_.get('path'), _.includes('tests-summary')), items);
     return _.map('url', testSummaryArtifacts);
-  } catch (e) {
-    console.error(`**  ERROR: Encountered error when getting CircleCI JOB_BUILD_NUM: ${buildNum} artifacts.`, e);
-    throw e;
+  } catch (error) {
+    console.error(`** ERROR fetching CircleCI JOB_BUILD_NUM: ${buildNum} artifacts.`);
+    console.error(error);
+    console.error(`HTTP error response body: ${await error.response.text()}`);
```

Comment:
This looks like it won't work... if the API returns a non-200 status, `checkStatus` throws an Error object. That object does not have a `response` property.